### ISSUE TITLE
Onychotillomania

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4778]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Habitual pick at nails or teaing them off.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding as HP_0032509, @MickeySegal
